HeroVault/HeroVault.xcodeproj/project.pbxproj

```diff
@@ -21,6 +21,8 @@
 		56B05DA82BD4057A001A0A61 /* MainViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 56B05DA72BD4057A001A0A61 /* MainViewController.swift */; };
```

Comment:
The name of your feature should have been feature/userProfile and not just userProfile. Please follow the guidelines more closely in the future😄

Reply:
Will do,  I'll rename the branch when the pr closes.

Reply:
no need to rename the branch. Just double check all future branches👍🏼